MAC_SETUP.md

```diff
@@ -102,22 +102,18 @@ test their code.
     ```
 
 ## Managing python versions using Pyenv global
-Pyenv allows you to configure the prioirty order for your python installs.
+Pyenv allows you to configure the priority order for your python installs.
 
 ```
 pyenv global 3.8.1 3.7.6 3.6.10 3.5.9 2.7.17
 ```
 
-This will make python and python3 point to Python 3.8.1. python2 will go to 2.7.17. You can also further specify versions, such as python3.6 to use that version.
+This will make python and python3 point to Python 3.8.1. python2 will use 2.7.17. You can also further specify versions, such as python3.6 to use that version.
 
 ## Python virtual environments
-Using Virtual Environments is fairly common across the Python community. While there is built in support in python for creating environments, there are also other tools that the community has embraced. Option 1 relies on python itself, Option 2 uses an additional tool that some users may prefer.
+Using [Virtual Environments](https://docs.python.org/3/library/venv.html) prevents inadvertent modifications to your global python install. Once created and sourced, calls to `python` will uses this virtual environment, not a global python install. Each virtual environment can have its own set of packages that can be different from others.
 
-In either case, you can create and source virtual environments, `python` will point to this virtual environment, not a global python install. Each virtual environment can have its own set of packages that can be different from others.
-
-
-### Option 1: Using builtin support, venv
-Python ships with support of its own to create virtual environments. You can [create a virtual environment](https://docs.python.org/3/library/venv.html).
+To create a virtual environment, run `python3 -m venv`.
 
 ```
 cd python-docs-samples
```

Comment:
perhaps putting venv-name in square brackets or uppercase to indicate it's something you fill in